schemas/telemetry/usage/browser-features-schema.json

```diff
@@ -6,12 +6,16 @@
   "description": "Schema of browser specific features usage",
   "oneOf": [
     {
-      "required": ["feature"],
+      "required": ["feature", "is_forced"],
       "properties": {
         "feature": {
           "type": "string",
           "description": "startSessionReplayRecording API",
           "const": "start-session-replay-recording"
+        },
+        "is_forced": {
+          "type": "boolean",
+          "description": "Whether the API was forced to record sampled out sessions"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** the phrasing seems a bit confusing to me, wdyt of
‏
```suggestion
          "description": "Whether the recording is allowed to start even on sessions sampled out of replay"
```